funnel/views/project.py

```diff
@@ -217,6 +217,19 @@ def csv(self):
             ],
         )
 
+    @route('rename', methods=['GET', 'POST'])
+    @requires_login
+    @requires_roles({'editor'})
+    def rename(self):
```

Comment:
This now "setting the name" more than "renaming". I think we should use something like "set a custom URL slug" or "customise the URL" ("slug" being a term from WordPress).